frontend/micro-ui/web/micro-ui-internals/packages/modules/hrms/src/pages/createEmployee.js

```diff
@@ -223,6 +223,17 @@ const CreateEmployee = () => {
       jurisdictions = mappedData;
     } else {
       roles = data?.Jurisdictions?.map((ele) => {
+        // Check if the roles array contains an object with code 'CHAIRMEN'
+        const hasChairmen = ele.roles.some((role) => role.code === "CHAIRMEN");
+        // If the roles array contains an object with code 'CHAIRMEN', push the DIV Admin role into the roles array
+        if (hasChairmen) {
+          ele.roles.push({
+            code: "HRMS_ADMIN",
+            name: "HRMS_ADMIN",
+            labelKey: "HRMS_ADMIN",
+            tenantId: data.testing,
```

Comment:
Did you tested for other tenant id also, apart from testing tenant @ShubhamDev056

Reply:
right now we have only testing tenant, so i am going to create a new user with different tenant then test pls wait while i am testing.